### PR TITLE
Right in table

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: WorkflowFunctionAdminBundle/Resources/views/Form/form_div_layout.html.twig [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% block workflow_right_widget %}
+    {% spaceless %}
+        <div class="table-responsive">
+            <table class="table table-bordered table-striped workflow-right-widget">
```
Comment (ngilain) [inline WorkflowFunctionAdminBundle/Resources/views/Form/form_div_layout.html.twig:4]: Can't you use something else than table ?

Comment (veolys-nanne) [inline WorkflowFunctionAdminBundle/Resources/views/Form/form_div_layout.html.twig:4]: table is used by smartadmin and ticket : (2) Etq Ubo, je vois les droits de workflow ss la forme d'un tableau à double entrée.
So there are always alternatives. But this solution is the best for me. @itkg-ngilain

**Comment by ngilain** [comment]
Ok if it's allready used by smartadmin => :+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**